files-community/Files
# Feature: Added shortcuts to switch between Preview & Detail panes

**Resolved / Related Issues**
   Closes #13679...

**Validation**
How did you test these changes?
   1. Use `ctrl` + `alt` + `d` to open/switch to the details pane
   2. Use `ctrl` + `alt` + `p` to open/switch to the preview pane
   3. Use the toolbar button to close the pane
   4. Confirm the toolbar button always restores the last selected pane

**Screenshots (optional)**
Add screenshots here.
